src/config-schema.js

```diff
@@ -1,68 +1,103 @@
-path = require 'path'
-fs = require 'fs-plus'
-
-# This is loaded by atom.coffee. See https://atom.io/docs/api/latest/Config for
```

Comment:
I think that link's still pointing to the right place – that seems to be the only place we document config schemas (even http://flight-manual.atom.io/behind-atom/sections/configuration-api/ just links there)